### DISCUSSION TITLE
Will there be an update to PHP 8.1.7?

### DISCUSSION THREAD

**Discussion by c1am** [php-mime-mail-parser/php-mime-mail-parser#396] [Q&A] [answered]
Hi, just wondering if there will be an update to make the parser compatible with PHP 8.1.7? Or is it already?

**Comment by eXorus** [comment, marked as answer]
Hello,

The parser is already compatible to PHP 8.1, see the changelog in the last release (from 7.1.0)
https://github.com/php-mime-mail-parser/php-mime-mail-parser/releases

I updated the readme.
